### DISCUSSION TITLE
About Chinese Student's applycation problems

### DISCUSSION THREAD

**Discussion by ImXuelang** [community/community#106644] [GitHub Education]
### Hi everyone,

Hi all, I'm having a bit of a problem applying for student certification. I am a Chinese high school student, and during the re-application process, I encountered that entering the full name of my school (Chinese and English) could not be recognized, and my school could not be found by the name of the school in the map (in fact, my school could be found on the Bing map and displayed as a high school), so I could only select the detailed address. I submitted my student ID, acceptance letter and my transcript, but because these are all in Chinese, they may not be recognized, what should I do? Our school doesn't have an edu email address for students, and we don't know what our school's website is.When my first application was rejected, GitHub prompted me to "Please specify school name in your next application."

**Comment by ghost** [comment]
我是连地图都打不开（哭笑不得）

**Comment by nick-cjyx9** [comment]
Hi, Xuelang,
> so I could only select the detailed address
1. Detailed address is enough usually, I'm not really know what do you mean.
> but because these are all in Chinese, they may not be recognized, what should I do?
2. You can put their translation next to them
> Our school doesn't have an edu email address for students, and we don't know what our school's website is.
3. The website isn't a must, you can also use baidubaike insteadly.Without a .edu email,you can also prove through the live photo of these documents.
> Please specify school name in your next application.
4. Sry that I can't recognize the reason behind the rejection upon your descriptions.

Hope can help you.
